blueoil/cmd/output_event.py

```diff
@@ -63,32 +74,40 @@ def output(tensorboard_dir, output_dir, metrics_keys, steps, output_file_base="m
 
     columns = [_column_name(event_accumulator, metrics_key)
                for event_accumulator, metrics_key in itertools.product(event_accumulators, metrics_keys)]
-    columns.sort()
-    df = pd.DataFrame([], columns=columns)
+
+    value_matrix = []
+
+    for metrics_key in metrics_keys:
+        if not value_matrix:
+            step_list = sorted(_step_list(event_accumulator, metrics_key), reverse=True)
+            value_matrix.append(step_list)
```

Comment:
With these lines, the list of steps comes from the first metrics_key. Is that the same with the original code? Did you check the result is the same?

Reply:
@tfujiwar 
Yes, I have verified the result.
The list of steps is the same and is repeated for each metrics_key, it is enough to just read it once from any metrics_key.

Reply:
OK. Then what about this code? I think we don't need to use a for-loop.
```py
value_matrix = [sorted(_step_list(event_accumulator, metrics_keys[0]), reverse=True)]
```

Reply:
@tfujiwar 
This can't work, there will be `TypeError: 'set' object is not subscriptable` because `metrics_keys` is a set.
Can I make it this way? (use `break` to break the for loop).
```   
for metrics_key in metrics_keys:
        step_list = sorted(_step_list(event_accumulator, metrics_key), reverse=True)
        value_matrix.append(step_list)
        break
```

Reply:
Hmm, I've understood the situation. It might be better to refactor the existing code also but this PR is OK 👍